src/foam/nanos/u2/navigation/SignIn.js

```diff
@@ -33,13 +33,11 @@ foam.CLASS({
         class: 'foam.u2.TextField',
         focused: true
       },
-      preSet: function(o, n) {
-        return n.trim();
-      },
       visibilityExpression: function(disableEmail_) {
         return disableEmail_ ?
           foam.u2.Visibility.DISABLED : foam.u2.Visibility.RW;
-      }
+      },
+      validationTextVisible: false
```

Comment:
So this applies to sign-up as well?

Reply:
I haven't looked into it. I will change signUp as well if you want me to. I can also check any other places that use EMail to make sure they work well.

Reply:
got it, thought signup and signin were the same model. So this is fine.  Thanks.